cmd/root.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/spf13/viper"
+	"github.com/veraison/evcli/cmd/psa"
+)
+
+var (
+	cfgFile   string
+	validArgs = []string{"psa"}
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:           "evcli",
+	Short:         "Attestation Evidence swiss-army knife",
```

Comment:
Is there a Long Version of the command. If present I suggest we add a bit more in detail what capabilities are provided by evcli.

Reply:
There is no need to repeat ourselves, cobra will to that for us.